src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArr = [];
+  let newBox = { ...state };
```

Comment:
I think u may to name it better

Reply:
Agree with Artem, let's use better naming for variables